temporian/core/test/event_set_ops_test.py

```diff
@@ -191,10 +191,6 @@ def test_propagate(self):
             )
```

Comment:
I see that we also have a `test_propagate` here, we should remove it too, right?